deepinv/datasets/div2k.py

```diff
@@ -30,65 +33,60 @@ class DIV2K(torch.utils.data.Dataset):
     }
 
     def __init__(
-        self, root: str, mode: str = "train", download: bool = False, transform=None
+        self,
+        root: str,
+        mode: str = "train",
+        download: bool = False,
+        transform: Callable = None,
     ) -> None:
-        super().__init__()
-
         self.root = root
         self.mode = mode
         self.transform = transform
         train_dir_path = os.path.join(self.root, "DIV2K_train_HR")
         valid_dir_path = os.path.join(self.root, "DIV2K_valid_HR")
 
-        # verify that dataset is available at self.root and use it
-        if self._check_dataset_exists():
-            print(
-                f"""
-            Dataset is available at `{self.root}`.
-            `download` flag is not taken into account.
-            """
-            )
-        # otherwise we try to download the whole dataset
-        elif download:
-            if not os.path.isdir(self.root):
-                os.makedirs(self.root)
-            if os.path.exists(train_dir_path):
-                raise ValueError(
-                    f"""
-                The train folder already exists,
-                thus the download is aborted.
-                Please set `download=False`
-                OR remove `{train_dir_path}`."""
-                )
-
-            if os.path.exists(valid_dir_path):
-                raise ValueError(
+        # download dataset, we check first that dataset isn't already downloaded
+        if not self._check_dataset_exists():
+            if download:
+                if not os.path.isdir(self.root):
+                    os.makedirs(self.root)
+                if os.path.exists(train_dir_path):
+                    raise ValueError(
+                        f"""
+                    The train folder already exists,
+                    thus the download is aborted.
+                    Please set `download=False`
+                    OR remove `{train_dir_path}`."""
+                    )
+
+                if os.path.exists(valid_dir_path):
+                    raise ValueError(
+                        f"""
```

Comment:
I think there is a newline in the string that should be removed 

f"""text here 

instead of

f""" (new line)
text here